cmd/bblfsh-sdk-tools/cmd/fixtures.go

```diff
@@ -62,72 +71,89 @@ func (c *FixturesCommand) generateFixtures(filename string) error {
 		return err
 	}
 
-	native, err := c.getNative(source, filename)
+	//UASTv2 sematic, native
+	err = c.writeUASTv2(source, filename, c.ExtNative, protocol2.Mode_Native)
 	if err != nil {
 		return err
 	}
 
-	err = c.writeResult(filename, native, c.ExtNative)
+	err = c.writeUASTv2(source, filename, c.ExtUast, protocol2.Mode_Semantic)
```

Comment:
This can be an option, so a user can select `Annotated` instead. For now, it makes sense to support both.

Reply:
Does it mean you suggest adding option for writing `.uast` in annotated mode as well?